e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/to-do-amended-actions/request-correction-for-corrected-fee.spec.js

```diff
@@ -0,0 +1,90 @@
+import {
+  FEE_RECORD_STATUS,
+  FeeRecordEntityMockBuilder,
+  UtilisationReportEntityMockBuilder,
+  PENDING_RECONCILIATION,
+  CURRENCY,
+  FeeRecordCorrectionEntityMockBuilder,
+  RECORD_CORRECTION_REASON,
+} from '@ukef/dtfs2-common';
+import pages from '../../../../../pages';
+import USERS from '../../../../../../fixtures/users';
+import { NODE_TASKS } from '../../../../../../../../e2e-fixtures';
+import { getMatchingTfmFacilitiesForFeeRecords } from '../../../../../../support/utils/getMatchingTfmFacilitiesForFeeRecords';
+
+context('When fee record correction feature flag is enabled', () => {
+  context('PDC_RECONCILE users can remove a corrected fee from a payment', () => {
```

Comment:
Needs updating to match the filename please, e.g.
```suggestion
  context('PDC_RECONCILE users can request a correction for a corrected fee', () => {
```